Yatima/CodeGen/CodeGen.lean

```diff
@@ -315,8 +315,7 @@ def codeGen (leanEnv : Lean.Environment) (decl : Name) : Except String Expr :=
   match CodeGenM.run ⟨leanEnv.patchUnsafeRec, .empty⟩ default (codeGenM decl) with
   | .error e _ => .error e
   | .ok _ s => do
-    let bindings := Expr.mutualize $
```

Comment:
`Expr.mutualize` is no longer necessary because `letrec` is alredy mutually recursive in current Lurk